src/ess/sans/workflow.py

```diff
@@ -99,8 +99,8 @@ def _set_runs(
     pipeline = pipeline.copy()
     runs = pd.DataFrame({Filename[key]: runs}).rename_axis(axis_name)
     for part in (Numerator, Denominator):
-        pipeline[CleanSummedQ[key, part]] = (
-            pipeline[CleanSummedQ[key, part]]
+        pipeline[WavelengthScaledQ[key, part]] = (
```

Comment:
I guess my only question is: did the results change at all after this modification?